cookbooks/podman/resources/service.rb

```diff
@@ -41,9 +41,17 @@
     restart "on-failure"
   end
 
+  # No start action here to avoid a start and then immediate restart on first run
   service new_resource.service do
-    action [:enable, :start]
-    subscribes :restart, "systemd_service[#{new_resource.service}]"
+    action :enable
+    supports :status => true, :restart => true
```

Comment:
This line isn't needed, and won't do anything when systemd is the service manager.